Server/Controllers/HelperController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Endava_Project.Server.Data;
+using Endava_Project.Server.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Endava_Project.Server.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class HelperController : ControllerBase
+    {
+        private readonly ApplicationDbContext context;
+
+        public HelperController(ApplicationDbContext context)
+        {
+            this.context = context;
+        }
+
+        [HttpGet]
+        public List<string> GetUsers()
+        {
```

Comment:
I already left a few comments for this method to Ecaterina Herta. I see quite a lot of similarities in your code, so please talk to each other and discuss the issues.